iCub/ros/rviz/iCubRviz.rviz

```diff
@@ -0,0 +1,733 @@
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_lower_leg_skin_49:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_lower_leg_skin_5:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_lower_leg_skin_50:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_lower_leg_skin_51:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_lower_leg_skin_52:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_lower_leg_skin_53:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_lower_leg_skin_54:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_lower_leg_skin_55:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_lower_leg_skin_56:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_lower_leg_skin_6:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_lower_leg_skin_60:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_lower_leg_skin_61:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_lower_leg_skin_9:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_shoulder_1:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+          Value: true
+        r_shoulder_2:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+          Value: true
+        r_shoulder_3:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+          Value: true
+        r_sole:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_upper_arm:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+          Value: true
+        r_upper_leg:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+          Value: true
+        r_upper_leg_back_contact:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_upper_leg_dh_frame:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        r_wrist_1:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+          Value: true
+        root_link:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+          Value: true
+        root_link_imu_frame:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+        torso_1:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+          Value: true
+        torso_2:
+          Alpha: 1
+          Show Axes: false
+          Show Trail: false
+          Value: true
```

Comment:
Not all the icub models have  all those links. Are all this options  compulsory? Can we just omit them?

Reply:
This is again the result of autogenrated `.rviz` configuration file as there is a robot model added inside rviz. I tested it with `iCubGenova02` and `iCubGazeboV2_5` urdf models.

Reply:
Yes, but what happens if somebody uses it with the `iCubGenova01` that for example does not contain the `l_forearm_skin_7` URDF "link"? What happens if in the future we add or remove links from the model? I would strongly prefer to have a conf file that we do not need to update for each change in the iCub model, and if it is not possible we should document how this file can be regenerated (if it cannot be regenerated automatically). 
However, according to http://wiki.ros.org/rviz/DisplayTypes/RobotModel this option seems to be optional, so I think it should be possible to just remove them without any particular problem (also simplifying sensibly the conf file).

Reply:
> However, according to http://wiki.ros.org/rviz/DisplayTypes/RobotModel this option seems to be optional, so I think it should be possible to just remove them without any particular problem (also simplifying sensibly the conf file).

Yes this option is not strictly necessary but the idea of having a ros package with a launch file and rviz configuration is that some one without much experience with ros or rivz can simple launch it without much hassle to add the robot model and then add the proper [prefix](https://github.com/Yeshasvitvs/icub-models/blob/7347e37aa715c27e8eecbaa90b82e5c42ca41c82/iCub/ros/rviz/iCubRviz.rviz#L664), which is set in `yarprobostatepublisher` parameters. I believe in this case it is better to leave a model in the rviz configurations as at least the users who will be using the rviz visualization tool actively are mostly (at least for the time being) users of latest models (DIC members).

Reply:
Is it enough to put description and instructions in a readme file so that other users can modify it if required? One of the main goals of this PR is the simplification of launching demos. Having the model already there makes it faster and smoother.

Reply:
Sorry, I did not want to remove the model. I just wanted to remove the optional per-link configuration, such as: 
~~~
        base_link:
          Alpha: 1
          Show Axes: false
          Show Trail: false
~~~
that can create problems if a different model is loaded. Do you think this can create problems in any way?